Source/Core/VideoCommon/VideoCommon.vcxproj.filters

```diff
@@ -176,9 +173,6 @@
     <ClCompile Include="AbstractTexture.cpp">
       <Filter>Base</Filter>
     </ClCompile>
-    <ClCompile Include="AbstractRawTexture.cpp">
```

Comment:
was it intentional for this to be removed (along with `TextureConvertionShaderGen` being corrected). Seems like something that should be in a separate PR.

Reply:
Hmm, yep, looks like VS auto-removed it since the file no longer exists. I'll make a separate PR.